cmd/internal/search_batch.go

```diff
@@ -27,12 +26,7 @@ import (
 	"go4.org/syncutil"
 )
 
-var (
-	flagMinNameScore = flag.Float64("min-match", 0.90, "How close must names match")
-	flagRequestID    = flag.String("request-id", "", "Override what is set for the X-Request-ID HTTP header")
-	flagSdnType      = flag.String("sdn-type", "individual", "sdnType query param")
-	flagThreshold    = flag.Float64("threshold", 0.99, "Minimum match percentage required for blocking")
-)
+var matchThreshold float64 = 0.99
```

Comment:
Why is this necessary to be out here? This seems like it could create race conditions if this endpoint handles simultaneous requests.